spec/support/helpers.rb

```diff
@@ -3,11 +3,12 @@ def stub_http(options = {})
     response = options[:response] || Net::HTTPSuccess.new('1.2', '200', 'OK')
     allow(response).to receive(:body).and_return(options[:body] || '{"id":"1234"}')
     http = double(:post          => response,
-                :read_timeout= => nil,
-                :open_timeout= => nil,
-                :ca_file=      => nil,
-                :verify_mode=  => nil,
-                :use_ssl=      => nil)
+                  :get           => response,
+                  :read_timeout= => nil,
+                  :open_timeout= => nil,
+                  :ca_file=      => nil,
```

Comment:
Probably should indent this hash rocket. I am nitpicking, but if we're fixing indentation we might as well. :)

Reply:
This is actually indented, not sure why Github is showing a different indentation :)
Check https://github.com/minhajuddin/honeybadger-ruby/blob/279073fff9d87f04fffd63d8ec0ce3406d9129b1/spec/support/helpers.rb#L5